src/screens/signup/signup2.tsx

```diff
@@ -0,0 +1,92 @@
+import React, { useState } from 'react';
+import { TouchableOpacity } from 'react-native';
+import { Ionicons } from '@expo/vector-icons';
+
+import { NavigationInterface } from '../types';
+import { useThemeContext } from '../../theme';
+
+import SafeAreaView from '../../commons/header/safe-area-view';
+import Button from '../../components/button';
+import Input from '../../components/input';
+
+import {
+  Container,
+  Welcome,
+  Prompt,
+  NavigationView,
+  Pagination,
+  OfText
+} from './styles';
+
+interface SignupScreenTwoProp extends NavigationInterface {
+  testID?: string;
+}
+
+export default function SignUpScreenTwo(props: SignupScreenTwoProp) {
+  const { colors, fonts } = useThemeContext();
+
+  const [email, setEmail] = useState('clairetamara08@gmail.com');
```

Comment:
Please merge all these state into one like this:
```suggestion
  const [state, setState] = useState({email: '',firstName: '', lastName:''});
```

Reply:
I have made the neccessary changes